README.md

```diff
@@ -77,9 +80,9 @@ Also see:
 
 ```bash
 # Preferred
-$ rake
-
-# Run all the tests without automatically starting jetty.
+$ rake # Run all tests. 
+  
+# Run all the tests including starting/stopping jetty and refreshing all fixtures.
```

Comment:
looks like it does in the definition of the task as it runs several other rake tasks including the ci one: https://github.com/sul-dlss/hydrus/blob/master/lib/tasks/ci.rake#L33